Client/qtTeamTalk/mainwindow.cpp

```diff
@@ -3815,7 +3815,7 @@ void MainWindow::slotClientPreferences(bool /*checked =false */)
     if((TT_GetFlags(ttInst) & CLIENT_AUTHORIZED) &&
         TT_GetUser(ttInst, TT_GetMyUserID(ttInst), &myself))
     {
-        QString nickname = ttSettings->value(SETTINGS_GENERAL_NICKNAME, QCoreApplication::translate("MainWindow", SETTINGS_GENERAL_NICKNAME_DEFAULT)).toString();

```

Comment:
Keep SETTINGS_GENERAL_NICKNAME_DEFAULT but set SETTINGS_GENERAL_NICKNAME_DEFAULT to an empty string